src/System.Runtime.Extensions/tests/Performance/Perf.Environment.cs

```diff
@@ -65,5 +66,105 @@ public void ExpandEnvironmentVariables()
                 Environment.SetEnvironmentVariable(env, null);
             }
         }
+
+        [Benchmark]
+        public void GetEnvironmentVariables()
+        {
+            PerfUtils utils = new PerfUtils();
+            string env = utils.CreateString(15);
+            try
+            {
+                // setup the environment variable so we can read it
+                Environment.SetEnvironmentVariable(env, "value");
+
```

Comment:
@DrewScoggins, the harness throws away iteration zero as a warm-up iteration, so unless the test is huge, the code should be hot by iteration one.

@jorive - can you confirm that I'm telling the truth.